src/OpeningHoursModel.h

```diff
@@ -13,7 +13,7 @@ class OpeningHoursModel : public QStringListModel
 {
     Q_OBJECT
 
-    Q_PROPERTY(QJSValue restaurant READ restaurant WRITE setRestaurant)
+    Q_PROPERTY(QJSValue openingData READ  openingData WRITE setOpeningData)
```

Comment:
there is a space too much in front of ` openingData`.

Reply:
done